<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZiyaretEkle</class>
 <widget class="QDialog" name="ZiyaretEkle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>407</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>515</width>
    <height>317</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Yeni Ziyaret Oluştur</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QTextEdit" name="teSikayet"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblHasta">
       <property name="text">
        <string>&amp;Hasta</string>
       </property>
       <property name="buddy">
        <cstring>hastaComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QDateTimeEdit" name="dteZiyaret">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QTextEdit" name="teTani">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblSikayet">
       <property name="text">
        <string>&amp;Şikayet</string>
       </property>
       <property name="buddy">
        <cstring>teSikayet</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblZiyaretTarihi">
       <property name="text">
        <string>&amp;Ziyaret Tarihi</string>
       </property>
       <property name="buddy">
        <cstring>dteZiyaret</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblTani">
       <property name="text">
        <string>&amp;Tanı</string>
       </property>
       <property name="buddy">
        <cstring>teTani</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="hastaComboBox"/>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QComboBox" name="doctorComboBox"/>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lblTedaviNotlari">
       <property name="text">
        <string>&amp;Tedavi Notları</string>
       </property>
       <property name="buddy">
        <cstring>teTedaviNotlari</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="teTedaviNotlari"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblDoktor">
       <property name="text">
        <string>&amp;İlgilenen Doktor</string>
       </property>
       <property name="buddy">
        <cstring>doctorComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnKaydet">
         <property name="text">
          <string>Kaydet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnVazgec">
         <property name="text">
          <string>Vazgeç</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>hastaComboBox</tabstop>
  <tabstop>doctorComboBox</tabstop>
  <tabstop>teSikayet</tabstop>
  <tabstop>dteZiyaret</tabstop>
  <tabstop>teTani</tabstop>
  <tabstop>teTedaviNotlari</tabstop>
  <tabstop>btnKaydet</tabstop>
  <tabstop>btnVazgec</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnKaydet</sender>
   <signal>clicked()</signal>
   <receiver>ZiyaretEkle</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnVazgec</sender>
   <signal>clicked()</signal>
   <receiver>ZiyaretEkle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
